providers/gitee.go

```diff
@@ -0,0 +1,48 @@
+// create by vinson on 2020/4/10
+
+package providers
+
+import (
+	"context"
+	"github.com/drone/drone-go/drone"
+	"github.com/drone/go-scm/scm"
+	"github.com/drone/go-scm/scm/driver/gitee"
+	"github.com/drone/go-scm/scm/transport"
+	"net/http"
+)
+
+func GetGiteeFilesChanged(repo drone.Repo, build drone.Build, token string) ([]string, error) {
+	newctx := context.Background()
+	var client *scm.Client
+	var err error
+
+	client = gitee.NewDefault()
+
+	client.Client = &http.Client{
+		Transport: &transport.BearerToken{
+			Token: token,
+		},
+	}
+
+	var changes []*scm.Change
+	var _ *scm.Response
+
+	if build.Before == "" || build.Before == scm.EmptyCommit {
+		changes, _, err = client.Git.ListChanges(newctx, repo.Slug, build.After, scm.ListOptions{})
```

Comment:
Thanks for working on this!

Unfortunately git-scm does not currently have support for `ListChanges` (or `CompareChanges` below) https://github.com/drone/go-scm/blob/master/scm/driver/gitea/git.go#L76-L78

This is preventing us from supporting Gitea as well (see #81).

I think you will need to add that support to the git-scm module before this extension can support Gitee.